test/source/tests/browser-unit-tests/unit-Mime.js

```diff
@@ -338,3 +338,24 @@ BROWSER_UNIT_TEST_NAME(`Mime attachment file name issue 3352`);
   }
   return 'pass';
 })();
+
+BROWSER_UNIT_TEST_NAME(`Mime attachment file name issue #3505`);
+(async () => {
+  const originalName = 'what\'s_up?.txt';
```

Comment:
@rrrooommmaaa FYI I tried to combine those duplicated code (With above logic) but then I realized I couldn't add custom javascript function in this file as this file is parsed in unit-browser.ts file.

Reply:
To re-use library code in unit browser tests we should add import into `extension/chrome/dev/ci_unit_test.ts` file and also add the imported instance to `libs` array there, if that's what you're asking.

Reply:
Aha, got it
Thanks